label_studio/frontend/src/pages/Settings/GeneralSettings.js

```diff
@@ -24,60 +29,95 @@ export const GeneralSettings = () => {
   ];
 
   const samplings = [
-    {value: "Sequential", label: "Sequential", description: "Tasks are ordered by Data manager ordering"},
-    {value: "Uniform", label: "Random", description: "Tasks are chosen with uniform random"},
+    { value: "Sequential", label: "Sequential", description: "Tasks are ordered by Data manager ordering" },
+    { value: "Uniform", label: "Random", description: "Tasks are chosen with uniform random" },
   ];
 
   return (
-    <div style={{width: 480}}>
-      <Form
-        action="updateProject"
-        formData={{...project}}
-        params={{pk: project.id}}
-        onSubmit={updateProject}
-      >
-        <Form.Row columnCount={1} rowGap="32px">
-          <Input
-            name="title"
-            label="Project Name"
-            labelProps={{large: true}}
-          />
+    <Block name="general-settings">
+      <div style={{ width: 488 }}>
```

Comment:
since BEM is being used this would be a good opportunity to update this native div to an Elem (if its still needed) ad update the css so we dont add inline styles here unless you feel this is too big a lift for this ticket and it should be addressed in another ticket

Reply:
You're right @yyassi-heartex, I fixed, thanks!